lib/model/rwfolder.go

```diff
@@ -139,29 +139,29 @@ func newRWFolder(m *Model, shortID protocol.ShortID, cfg config.FolderConfigurat
 		errorsMut: sync.NewMutex(),
 	}
 
-	configureCopiersAndPullers(folder, cfg)
+	folder.configureCopiersAndPullers(cfg)
 
 	return folder
 }
 
-func configureCopiersAndPullers(folder *rwFolder, config config.FolderConfiguration) {
-	if folder.copiers == 0 {
-		folder.copiers = defaultCopiers
+func (f *rwFolder) configureCopiersAndPullers(config config.FolderConfiguration) {
```

Comment:
I would leave it for later. When reading `configure` I would step into to see, what is going to be configured.